README.md

```diff
@@ -1 +1,175 @@
-# CodersCrew Website and Members Panel
\ No newline at end of file
+# CodersCrew Website and Members Panel
+
+[![Blitz.js](https://raw.githubusercontent.com/blitz-js/art/master/github-cover-photo.png)](https://blitzjs.com)
+
+This is a [Blitz.js](https://github.com/blitz-js/blitz) app.
+
+# **CodersCrewApp**
```

Comment:
This title shouldn't be here as our project title is defined at the top of the README.